hack/bundle-gen.py

```diff
@@ -186,7 +186,10 @@ def get_previous_version(channel_name):
         raise
 
 
-def generate_csv_base(bundle_dir, version, prev_version, channel):
+def generate_csv_base(hive_repo_dir, bundle_dir, version, prev_version, channel):
+    print("Working in {}".format(hive_repo_dir))
+    os.chdir(hive_repo_dir)
```

Comment:
I think this actually needs to go above the checkout on L552. (Otherwise we're landing in the default branch in the temp clone.)

Reply:
Moved!

Reply:
I squashed again to add comments about CWD.